service/controller/resource/tccpn/create.go

```diff
@@ -458,16 +459,8 @@ func (r *Resource) newIAMPolicies(ctx context.Context, cr infrastructurev1alpha3
 						return nil, microerror.Mask(err)
 					}
 
-					// First try using the alias domain (irsa.bla12.k8s. ... ).
-					cloudfrontDomain = cm.Data["domainAlias"]
-					if cloudfrontDomain == "" {
-						// Default to cloudfront domain if there is no alias.
-						cloudfrontDomain = cm.Data["domain"]
-						if cloudfrontDomain == "" {
-							return nil, microerror.Maskf(emptyDataError, "domain value in irsa cloudfront configmap for cluster must not be empty")
-						}
-					}
-
+					cloudfrontDomain = cm.Data["domain"]
```

Comment:
Not sure if this is right, in any case if the irsa-operator doesn't create the config map properly for handling the cloudfront part and the aws-operator is doing a lookup to get the values from non-existing keys, this will panic.

Reply:
The aws-operator could for instance reconcile the cluster before the irsa-operator notices the cluster, as an example.

Reply:
fair enough